### DISCUSSION TITLE
How to exclude /actuator/health endpoint span

### DISCUSSION THREAD

**Discussion by alvin562** [open-telemetry/opentelemetry-java-instrumentation#6597] [Q&A]
We have a java service that we automatically instrumented with opentelemetry and everythings worked pretty well. However, one issue we have is we are seeing a lot of spans to /actuator/health endpoint (essentially a health check from our ELB) coming from the tomcat-7.0 library. Does the java instrumentation provide any way we can exclude these spans from being emitted?

We tried looking into specific properties/env variables but couldn't find anything. Any feedback would be greatly appreciated. Thanks!

**Comment by trask** [comment]
hi @alvin562! check out #1060
